package.json

```diff
@@ -58,13 +61,13 @@
     "docs:prepare": "lerna run docs && pnpm retype:updateversion",
     "docs:dev": "retype watch .",
     "retype:updateversion": "sed \"s/label: .*/label: $(cat packages/cli/package.json | jq -r '.version')/\" retype.yml > retype.yml.tmp && mv retype.yml.tmp retype.yml",
-    "test": "pnpm recursive run test",
+    "test": "pnpm nx run-many --target=test",
     "entry:dist": "lerna run prepack",
     "entry:src": "lerna run postpack",
-    "gas-report": "pnpm recursive run gas-report",
+    "gas-report": "pnpm nx run-many --target=gas-report",
     "codegen": "pnpm codegen:tablegen && pnpm codegen:worldgen",
-    "codegen:tablegen": "pnpm recursive run tablegen",
-    "codegen:worldgen": "pnpm recursive run worldgen"
+    "codegen:tablegen": "pnpm nx run-many --target=tablegen",
+    "codegen:worldgen": "pnpm nx run-many --target=worldgen"
```

Comment:
nx has a lot of features but I tend to only use 3 day to day

`nx build` or `nx foo` is the same as `pnpm foo` but nx is smart enough to do things the script depends on first (like build sub dependencies) if it needs to.  Otherwise skip.  And it's shorter to type than pnpm foo.

`nx run-many --targets=foo,bar,baz` runs foo script bar script and baz script in parallel in every package

`nx affected foo` runs foo in only packages that have changed since main branch

Reply:
do these need `pnpm` prefix?